README.md

```diff
@@ -177,6 +177,22 @@ class ApplicationController < ActionController::Base
 end
 ```
 
+If you're using `verify_authorized` in your controllers but need to
+conditionally bypass verification, you can use `skip_authorization`. This is
+useful in circumstances where you don't want to disable verification for the
+entire action, but have some cases where you intend to not authorize.
```

Comment:
I think a change like that would be better in a full documentation change, but for the purposes of this PR I'd like to remain consistent with the existing style.